testfiles/5Zone_Unitary_VSDesuperheater.idf

```diff
@@ -2696,7 +2696,7 @@
     Coil:Heating:Fuel,       !- Heating Coil Object Type
     Main Heating Coil 1,     !- Heating Coil Name
     ,                        !- DX Heating Coil Sizing Ratio
-    Coil:Cooling:DX:VariableSpeed,  !- Cooling Coil Object Type
+    Coil:Cooling:DX,         !- Cooling Coil Object Type
```

Comment:
So the name of this file is 5Zone_Unitary_VSDesuperheater, I guess you just switched over to a VS coil using the new coil model. Fine.